src/helpers.ts

```diff
@@ -427,3 +427,23 @@ export function tagsMatch(tags, pattern) {
   const matchess = micromatch(tags || ['*'], pattern);
   return matchess.length > 0;
 }
+
+export function replaceFileNameWithJourneyName({
```

Comment:
Why do we do this? Is this for to avoid duplicating the videos between each runs?

Reply:
because default videonames generated by playwright are random uuid strings, having a journeyname is helpful while viewing artifacts